### ISSUE TITLE
news app v5.2.8 is not working anymore after upgrading from ownCloud 8.0.4 to 8.0.5

### ISSUE THREAD

**Issue by j-ed** [owncloud/news#822]
## Expected behavior
The news app should show all configured news sources in ownCloud 8.0.5.

## Current behavior
The news app doesn't work anymore and a blanc screen is shown in ownCloud 8.0.5.
The following error message has been written to the log file:
```
...,"app":"PHP","message":"Call to undefined method OCP\\AppFramework\\Http\\TemplateResponse::setContentSecurityPolicy() at \/var\/www\/htdocs\/owncloud\/apps2\/news\/controller\/pagecontroller.php#82","level":3,"time":"2015-07-08 23:26:30+02:00","method":"GET","url":"\/owncloud\/index.php\/apps\/news\/"}
```
My assumption is that oC 8.0.5 already uses functions which have been applied in v5.3.0 for oC 8.1.
Is there any chance to get the news app up-and-running again with oC 8.0.5?

## Steps to reproduce
1. updated an oC 8.0.4 server with an installed news app v5.2.8
2. access the news app.

#### Server Configuration
OS: Linux 3.2.67
Web server: Apache2 2.4.12
Database: MySQL 5.1.73
PHP version: 5.4.42
ownCloud version: 8.0.5

#### Client Configuration
Browser: Firefox 39.0
Operating system: Windows 7

**Comment by Raydiation** [comment]
In general: There is no support for older version ;)

Your problem seems to happen because it finds a contentsecuritypolicy.php (https://github.com/owncloud/news/blob/5.2.8/controller/pagecontroller.php#L74) in here  https://github.com/owncloud/core/tree/stable8/lib/public/appframework/http which is only available in 8.1+ normally. 

Leftover file from a failed 8.1 upgrade?

**Comment by j-ed** [comment]
Thank you for your help, removing the mentioned `contentsecuritypolicy.php` has solved the problem.
I know that it is difficult to support different app versions for different ownCloud versions but ownCloud 8.1 is not yet working with all required apps as expected. A direct update from v8.0.4 has failed and I wasn't able to login anymore via the WebGUI. Therefore I decided to give v8.0.5 a try and to use it as base version for my next update attempt.
